mcc/CaseConversion.c

```diff
@@ -81,7 +81,8 @@ static void MangleCharacters(struct InstData *data, char (*change)(char c))
     startline = GetNextLine(startline);
   }
 
-  data->HasChanged = TRUE;
+  data->HasChanged  = TRUE;
```

Comment:
revert the added space before the `= TRUE;` part (also in other source code areas where you changed the data->HasChanged indentation. This is an unnecessary change.